app/src/components/Call/RemoteParticipant.tsx

```diff
@@ -5,7 +5,7 @@ export type Props = { title: string; subtitle: string };
 
 const RemoteParticipant: React.FC<Props> = ({ title, subtitle }) => {
   return (
-    <View style={styles.container}>
+    <View style={styles.container} accessibilityLabel="remote participant">
```

Comment:
should we capitalize?

Reply:
I wasn't sure actually, accessibility is something I'm not familiar with at all. Is the standard to capitalize every word?

Reply:
Not sure either. Can we remove it? I think accessibility is a big effort in general.

Reply:
I agree, it's a huge effort. Unfortunately, I don't think we can do any component testing without at least a little bit of labeling or hinting.

Reply:
Gotcha. We can leave it as is for now 👍